modules/tensor_mechanics/src/materials/ComputeElasticityTensorCP.C

```diff
@@ -29,6 +29,9 @@ ComputeElasticityTensorCP::ComputeElasticityTensorCP(const InputParameters & par
     _crysrot(declareProperty<RankTwoTensor>("crysrot")),
     _R(_Euler_angles)
 {
+  // the base class guarantees constant in time, but in this derived class the
+  // tensor might rotate over time
```

Comment:
Might is probably the wrong word here--once the crystal plasticity model hits plasticity, the lattice will be rotated